codespell_lib/_codespell.py

```diff
@@ -1178,20 +1178,24 @@ def main(*args: str) -> int:
                 dirs[:] = [dir_ for dir_ in dirs if not glob_match.match(dir_)]
 
         elif not glob_match.match(filename):  # skip files
-            bad_count += parse_file(
-                filename,
-                colors,
-                summary,
-                misspellings,
-                exclude_lines,
-                file_opener,
-                word_regex,
-                ignore_word_regex,
-                uri_regex,
-                uri_ignore_words,
-                context,
-                options,
-            )
+            if not os.path.exists(filename):
```

Comment:
Rather than a new conditional here, why not modify line 842 to have:
```
            print("ERROR: file does not exist: %s" % filename, file=sys.stderr)
            bad_count += 1
```
it seems like a better place to add the check

Reply:
@larsoner I had to rebase, so I am not certain which line 842 you are referring to. Perhaps this one?
https://github.com/codespell-project/codespell/blob/a239c801c7bdece04f564ecbbc9b88e8261b4cca/codespell_lib/_codespell.py#L849

In any case, the idea is that we report an error if the files passed as arguments do not exist. We are merely checking command line arguments here.